src/app/public/modules/progress-indicator/progress-indicator.component.spec.ts

```diff
@@ -181,29 +159,29 @@ describe('Progress indicator component', function () {
     });
 
     it('should not advance once past the final step', () => {
-      componentInstance.messageStream.next(SkyProgressIndicatorMessageType.Progress);
+      componentInstance.sendMessage(SkyProgressIndicatorMessageType.Progress);
       fixture.detectChanges();
-      componentInstance.messageStream.next(SkyProgressIndicatorMessageType.Progress);
+      componentInstance.sendMessage(SkyProgressIndicatorMessageType.Progress);
       fixture.detectChanges();
-      componentInstance.messageStream.next(SkyProgressIndicatorMessageType.Progress);
+      componentInstance.sendMessage(SkyProgressIndicatorMessageType.Progress);
       fixture.detectChanges();
 
-      expect(componentInstance.activeIndex).toBe(3);
-      expect(componentInstance.progressItems.last.isActive).toBeFalsy();
```

Comment:
It passed because the logic of the steps was incorrect. The final step has an index of `2` because there's only three items. The final step should also be marked as `active`. Let me know if I'm missing something obvious!